azurelinuxagent/common/rdma.py

```diff
@@ -133,14 +133,14 @@ def load_driver_module(self):
         """Load the kernel driver, this depends on the proper driver
            to be installed with the install_driver() method"""
         logger.info("RDMA: probing module '%s'" % self.driver_module_name)
-        result = shellutil.run('modprobe %s' % self.driver_module_name)
+        result = shellutil.run('modprobe --first-time %s' % self.driver_module_name)
```

Comment:
load_driver_module is only used when installing/updating drivers (install_rdma_drivers). If the RDMA driver is already loaded, it's wrong that we don't do anything, because the newly installed driver is not loaded at this point.